tests/e2e/svmotion_detached_volume.go

```diff
@@ -163,4 +169,268 @@ var _ = ginkgo.Describe("[csi-block-vanilla] [csi-block-vanilla-parallelized] Re
 		}
 		gomega.Expect(err).NotTo(gomega.HaveOccurred())
 	})
+
+	/* Online relocation of volume using cnsRelocate Volume API
+	STEPS:
+	1. Create a tag based policy with 2 shared vmfs datastores(sharedVmfs-0 and sharedVmfs-1).
+	2. Create SC with storage policy created in step 1.
+	3. Create a PVC with sc created in step 2 and wait for it to come to bound state.
+	4. Create a pod with the pvc created in step 3 and wait for it to come to Running state.
+	5. Relocate volume from one shared datastore to another datastore using
+	   CnsRelocateVolume API and verify the datastore of fcd after migration and volume compliance.
+	6. Delete pod,pvc and sc.
+	*/
+	ginkgo.It("Online relocation of volume using cnsRelocate Volume API", func() {
+		ginkgo.By("Invoking Test for offline relocation")
+		ctx, cancel := context.WithCancel(context.Background())
+		defer cancel()
+
+		sharedvmfsURL := os.Getenv(envSharedVMFSDatastoreURL)
+		if sharedvmfsURL == "" {
+			ginkgo.Skip(fmt.Sprintf("Env %v is missing", envSharedVMFSDatastoreURL))
+		}
+
+		sharedvmfs2URL := os.Getenv(envSharedVMFSDatastore2URL)
+		if sharedvmfs2URL == "" {
+			ginkgo.Skip(fmt.Sprintf("Env %v is missing", envSharedVMFSDatastore2URL))
+		}
+		datastoreUrls := []string{sharedvmfsURL, sharedvmfs2URL}
+
+		govmomiClient := newClient(ctx, &e2eVSphere)
+		pc := newPbmClient(ctx, govmomiClient)
+		scParameters := make(map[string]string)
+		pvcs := []*v1.PersistentVolumeClaim{}
+
+		rand.Seed(time.Now().UnixNano())
+		suffix := fmt.Sprintf("-%v-%v", time.Now().UnixNano(), rand.Intn(10000))
+		categoryName := "category" + suffix
+		tagName := "tag" + suffix
+
+		catID, tagID := createCategoryNTag(ctx, categoryName, tagName)
```

Comment:
Add ginkgo.By also some info messages to the attach tag and create category steps